README.rst

```diff
@@ -419,3 +419,22 @@ Authors
 License
 -------
 `sgqlc` is licensed under the `ISC <https://opensource.org/licenses/ISC>`_.
+
```

Comment:
Since I was at it, I thought I might just as well document it to make it easier for future people.